azure-pipelines.yml

```diff
@@ -678,6 +678,24 @@ jobs:
 
     displayName: 'Test Homebrew Package'
 
+
+- job: BuildYumPackageMariner
+  displayName: Build Yum Package Mariner
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+  - task: Bash@3
+    displayName: 'Build Rpm Package: Mariner'
+    inputs:
+      targetType: 'filePath'
+      filePath: scripts/release/rpm/pipeline_mariner.sh
+  - task: PublishPipelineArtifact@0
+    displayName: 'Publish Artifact: yum-mariner'
+    inputs:
+      TargetPath: $(Build.ArtifactStagingDirectory)
+      ArtifactName: yum-mariner
```

Comment:
To be honest, I dislike using `yum` and `rpm` interchangeably (as it already is). This makes it difficult to understand for newcomers.